dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -32,6 +32,17 @@ RUN apt-get update   && \
 {{ install_debian_packages(docker_snmp_sv2_debs.split(' ')) }}
 {%- endif %}
 
+{%- if CONFIGURED_ARCH == "armhf" %}
+# Fix for hiredis compilation issues for ARM
```

Comment:
in this dockerfile, hiredis compilation is done in the docker-slave, so I am not sure why we have compilation issue here in snmp docker build.

Reply:
Just below these lines "hiredis" python is being installed.

```
 49 # Install pyyaml dependency for use by some plugins
 50 # Install smbus dependency for use by some plugins
 51 RUN python3.6 -m pip install --no-cache-dir \
 52         hiredis                             \
 53         pyyaml                              \
 54         smbus
```

Reply:
i think this locales fixes should not be limited to armhf, it should be applied to all arch, and we'd better to put them under docker-base-stretch which is shared by every other docker.

here is what we do for the base image. I think we should do the same thing as in the docker-base-stretch.

```
#Adds a locale to a debian system in non-interactive mode
sudo sed -i '/^#.* en_US.* /s/^#//' $FILESYSTEM_ROOT/etc/locale.gen && \
    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT locale-gen "en_US.UTF-8"
sudo LANG=en_US.UTF-8 DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT update-locale "LANG=en_US.UTF-8"
sudo LANG=C chroot $FILESYSTEM_ROOT bash -c "find /usr/share/i18n/locales/ ! -name 'en_US' -type f -exec rm -f {} +"
```